index.js

```diff
@@ -13,133 +13,127 @@
- * 
+ *
  * A demo can be seen at:
  * https://astexplorer.net/#/gist/7bc4771564a12c9f93c4904b3934aa1c/latests
  */
 
-module.exports = function (babel) {
-    const { types: t } = babel;
-
-    return {
-        visitor: {
-            /**
-             * Find all JSX elements that use the old
-             * <glamorous.Something> notation and rewrite them to be
-             * plain ole somethings.
-             */
-            JSXElement(path) {
-                path.traverse({
-                    "JSXOpeningElement|JSXClosingElement"(path) {
-                        if (!t.isJSXMemberExpression(path.node.name)) {
-                            return;
-                        }
-                        if (path.node.name.object.name !== "glamorous") {
-                            return;
-                        }
-                        path.node.name = t.identifier(
-                            path.node.name.property.name.toLowerCase()
-                        );
-                    }
-                });
-            },
-
-            /**
-             * Find glamorous import statements, including ThemeProvider
-             * imports and rewrite them to be emotion imports.
-             */
-            ImportDeclaration(path, { opts }) {
-                if (path.node.source.value !== "glamorous") {
-                    return;
-                }
+module.exports = function(babel) {
+  const {types: t} = babel;
 
-                // First, check for ThemeProvider and update that.
-                const themeProviderIndex = path.node.specifiers.findIndex(
-                    specifier => specifier.local.name === "ThemeProvider"
-                );
+  // glamorous and emotion treat the css attribute "content" differently.
+  // we need to put it's content inside a string.
+  // i.e. turn {content: ""} into {content: '""'}
+  const fixContentProp = glamorousFactoryArguments => {
+    return glamorousFactoryArguments.map(arg => {
+      if (t.isObjectExpression(arg)) {
+        arg.properties = arg.properties.map(
+          prop =>
+            prop.key.name === "content"
+              ? {...prop, value: t.stringLiteral(`"${prop.value.value}"`)}
+              : prop
+        );
+      }
+      // TODO: if `arg` is a function, we might want to inspect its return value
+      return arg;
+    });
+  };
 
-                if (~themeProviderIndex) {
-                    path.insertAfter(
-                        t.importDeclaration(
-                            [
-                                t.importSpecifier(
-                                    t.identifier("ThemeProvider"),
-                                    t.identifier("ThemeProvider")
-                                )
-                            ],
-                            t.stringLiteral("emotion-theming")
-                        )
-                    );
-                }
+  const styledVisitor = {
```

Comment:
Can we document this a little better?

`name`, `oldName`, etc. seem a little ambiguous.